capten/common-pkg/plugins/git/git.go

```diff
@@ -124,3 +115,22 @@ func (g *GitClient) Push(branchName, token string) error {
 
 	return err
 }
+
+func (g *GitClient) IsEmptyChanges() (bool, error) {
+	w, err := g.repository.Worktree()
+	if err != nil {
+		return false, err
+	}
+
+	status, err := w.Status()
+	if err != nil {
+		return false, err
+	}
+
+	untracked := status.IsUntracked("") && len(status) == 1
+	if status.IsClean() || (!untracked && len(status) == 1) {
+		return true, errors.New("no commit changes found")

```

Comment:
(!untracked && len(status) == 1) 
why this check needed ?

it should have status count 0, need to igonre the untrack files   // Check for modified files
	status, err := worktree.Status()
	if err != nil {
		log.Fatal(err)
	}

	// Iterate through the status and check if there are any modified files
	hasModifiedFiles := false
	for _, fileStatus := range status {
		if fileStatus.Staging != git.Untracked {
			hasModifiedFiles = true
			break
		}
	}